azurerm/resource_arm_automation_account.go

```diff
@@ -140,6 +158,7 @@ func resourceArmAutomationAccountRead(d *schema.ResourceData, meta interface{})
 	}
 	resGroup := id.ResourceGroup
 	name := id.Path["automationAccounts"]
+	skuName := d.Get("sku_name").(string)
```

Comment:
What you can do is mark both as computed until 2.0 and just set both instead of doing tricky logic like this

Reply:
Would that cause issues when we totally remove `sku`?